site/docs/mark/errorbar.md

```diff
@@ -36,59 +39,59 @@ An error bar's mark definition contain the following properties:
 
 Besides the properties listed above, `rule` and `ticks` can be used to specify the underlying [mark properties](mark.html#mark-def) for different [parts of the error bar](#parts) as well.
 
-
 {:#raw-usage}
+
 ## Using Error Bars to Aggregate Raw Data
 
 If the data is not aggregated yet, Vega-Lite will aggregate the data based on the `extent` properties in the mark definition.
 
-
-1) __Error bars showing standard error__ is the default error bar in Vega-Lite. It can also be explicitly specified by setting `extent` to `"stderr"`. The length of lower and upper rules represent standard error. By default, the rule marks expand from the mean.
+1. **Error bars showing standard error** is the default error bar in Vega-Lite. It can also be explicitly specified by setting `extent` to `"stderr"`. The length of lower and upper rules represent standard error. By default, the rule marks expand from the mean.
 
 <div class="vl-example" data-name="layer_point_errorbar_2d_horizontal"></div>
 
-
-2) __Error bar showing standard deviation__ can be specified by setting `extent` to `"stdev"`. For this type of error bar, the length of lower and upper rules represent standard deviation. Like an error bar that shows Standard Error, the rule marks expand from the mean by default.
+2. **Error bar showing standard deviation** can be specified by setting `extent` to `"stdev"`. For this type of error bar, the length of lower and upper rules represent standard deviation. Like an error bar that shows Standard Error, the rule marks expand from the mean by default.
```

Comment:
Not sure what you're doing, but I like __ more than ** lol

Reply:
I think it is the prettier again...

But this branch is just for testing why github does not delete .svg, .vg.json, and normalized.vl.json that we no longer use.

`check-and-fix.sh` calls `yarn build:examples`. And, `yarn build:examples` calls `yarn build-examples.sh`, which runs:
```
rm -f examples/specs/normalized/*_normalized.vl.json
rm -f $dir/*.vg.json
```
So, I'm trying to see why this is not working.